docs/docs/configuration/animations.mdx

```diff
@@ -133,18 +133,20 @@ A custom mode can be used by passing a custom `mode` to [update](../developers/a
 A mode option is defined by the same options of the main [animation configuration](#animation-configuration).
 
 ### Default modes
+The animation mode configuration can be adjusted in the `options.animation` namespace.
```

Comment:
nit: should have a empty line between heading and content (there are more of these, not commenting on all)